tests/test-helper.js

```diff
@@ -1,7 +1,8 @@
 import Application from '../app';
+import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
 
-setApplication(Application.create({ autoboot: false }));
+setApplication(Application.create(config.APP));
```

Comment:
I think this was causing the failed tests, IIRC some ember-cli 2.17 blueprint files were not working correctly with the new testing API. Had this in another repo, upgrading to ember-cli 2.18 fixed it (with `ember-cli-update`, so updated blueprint files would also be applied). But hey, why not go directly to 3.0? ;)

Reply:
Might as well 😄